src/middleware/index.ts

```diff
@@ -1,14 +1,31 @@
 import { Request, Response, NextFunction } from "express";
-import { ERRORS, scrubTokens } from "../utils";
-import { validDateTime } from "../utils/datetime";
-import { getProvider } from "../domains/providers";
-import { getCollections } from "../domains/collections";
+
+import { Provider } from "../models/CmrModels";
 import { StacQuery } from "../models/StacModels";
-import { InvalidParameterError } from "../models/errors";
+import {
+  InvalidParameterError,
+  ItemNotFound,
+  ServiceUnavailableError,
+} from "../models/errors";
+
+import { WarmCache } from "../domains/cache";
+import { getCollections } from "../domains/collections";
 import { parseOrdinateString } from "../domains/bounding-box";
+import { getProviders, getCloudProviders } from "../domains/providers";
 
-const MAX_LIMIT_SIZE = 250;
+import { scrubTokens, mergeMaybe, ERRORS } from "../utils";
+import { validDateTime } from "../utils/datetime";
+import { uniq } from "lodash";
+
+const STAC_QUERY_MAX = 5000;
```

Comment:
Determined by stress testing with UAT holdings

Reply:
is that worth a comment so that future developers don't change this without similar do diligence?